pubmed_dl/main.py

```diff
@@ -1,18 +1,23 @@
-#!/usr/bin/env python
 import json
-from pubmed_dl.ncbi import get_list_pmid, uids_to_docs
+import requests
+import os
+import sys
+from ncbi import uids_to_docs , get_list_pmid
 
 def main():
     ab = open("test.json", 'w')
-    start = input("Enter start date (YYYY/MM/DD): ")
-    end = input("Enter end date (YYYY/MM/DD): ")
+    #start = input("Enter start date (YYYY/MM/DD): ")
+    #end = input("Enter end date (YYYY/MM/DD): ")
+    start = "2020/12/04"
+    end = "2020/12/07"
     val = get_list_pmid(start,end)
+    val[-1] = "bruh"
     here = uids_to_docs(val)
-    for item in here:
-        ab.write(json.dumps(item))
-        ab.write("\n")
+    for item in uids_to_docs(val):
+        ab.write(json.dumps(item) + '\n')
+

```

Comment:
```python
for batch in uids_to_docs(val):
    ab.write("\n".join([json.dumps(doc) for doc in batch])
```

Should fix your issue of writing an entire batch to one line. Test it though

Reply:
This works thanks